community/aqua/artificial_intelligence/qsvm_variational.ipynb

```diff
@@ -137,24 +159,105 @@
     }
    ],
    "source": [
-    "result = run_algorithm(params, algo_input)\n",
+    "result = run_algorithm(params, svm_input, backend=backend)\n",
     "print(\"testing success ratio: \", result['testing_accuracy'])\n",
     "print(\"predicted classes:\", result['predicted_classes'])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### part II: programming way.\n",
+    "We construct the svm instance directly from the classes. The programming way offers the users better accessibility, e.g., the users can access the internal state of svm instance or invoke the methods of the instance. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we create the svm in the programming way.\n",
+    "- we build the optimizer instance (required by the svm instance) by instantiating the class SPSA.\n",
+    "- We build the feature map instance (required by the svm instance) by instantiating the class SecondOrderExpansion.\n",
+    "- We build the varitional form instance (required by the svm instance) by instantiating the class RYRZ.\n",
+    "- We build the svm instance by instantiating the class QSVMVariational. \n"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "backend = get_aer_backend('qasm_simulator')\n",
+    "optimizer = SPSA(max_trials=10, c0=4.0, skip_calibration=True)\n",
```

Comment:
could we reproduce the result with only 10 iterations?

Reply:
yes, this is what we observed with 10 trials.